ansible/roles/basic_users/library/terminate_user_sessions.py

```diff
@@ -56,13 +56,11 @@ def run_module():
 
     _, sessions_stdout, _ = module.run_command("loginctl --no-legend list-sessions", check_rc=True)
```

Comment:
`loginctl` supports `--output=json-pretty` - parsing JSON might prevent a breaking format change happening again in the future?

Reply:
Should have commented - I found that but it doesn't seem to do anything though for list-sessions at least:
```
[rocky@patches-control ~]$ loginctl --output=json-pretty list-sessions
SESSION  UID USER  SEAT TTY   IDLE SINCE
      1 1000 rocky      pts/0 no        

1 sessions listed.
```

Reply:
The weird thing is that it's not listed in the options for `loginctl --help` but trying something like `loginctl --otuput=madeup` errors, so clearly it is parsing it.